homeassistant/components/yale_smart_alarm/__init__.py

```diff
@@ -22,16 +22,22 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     await coordinator.async_config_entry_first_refresh()
 
     hass.data[DOMAIN][entry.entry_id] = {
-        "coordinator": coordinator,
+        COORDINATOR: coordinator,
     }
```

Comment:
If the coordinator is the only thing ever stored in the `hass.data[DOMAIN][entry.entry_id]`, it can be stored directly instead of making a `dict`

Reply:
I agree but it would affect also other platforms. Should we then incorporate that here in this PR even if it's not about the lock platform itself?

Reply:
Let's leave it for a fresh pr